GVRf/Framework/framework/src/main/res/raw/phong_surface_layertex.fsh

```diff
@@ -27,36 +27,36 @@ layout(location = 8) in vec2 lightmap_coord;
 
 #ifdef HAS_ambientTexture1
 layout(location = 12) in vec2 ambient_coord1;
-layout(set = 0, binding = 12) uniform sampler2D ambientTexture1;
+layout(set = 0, binding = 13) uniform sampler2D ambientTexture1;
 #endif
 
 #ifdef HAS_diffuseTexture1
 layout(location = 11) in vec2 diffuse_coord1;
-layout(set = 0, binding = 11) uniform sampler2D diffuseTexture1;
+layout(set = 0, binding = 12) uniform sampler2D diffuseTexture1;
 #endif
 
 #ifdef HAS_specularTexture1
 layout(location = 13) in vec2 specular_coord1;
-layout(set = 0, binding = 13) uniform sampler2D specularTexture1;
+layout(set = 0, binding = 14) uniform sampler2D specularTexture1;
 #endif
 
 #ifdef HAS_emissiveTexture1
 layout(location = 14) in vec2 emissive_coord1;
-layout(set = 0, binding = 14) uniform sampler2D emissiveTexture1;
+layout(set = 0, binding = 15) uniform sampler2D emissiveTexture1;
 #endif
 
 #ifdef HAS_lightMapTexture1
 in vec2 lightmap_coord1;
 #endif
 
 
-layout(set = 0, binding = 5) uniform sampler2D ambientTexture;
-layout(set = 0, binding = 4) uniform sampler2D diffuseTexture;
-layout(set = 0, binding = 6) uniform sampler2D specularTexture;
-layout(set = 0, binding = 7) uniform sampler2D opacityTexture;
-layout(set = 0, binding = 8) uniform sampler2D lightmapTexture;
-layout(set = 0, binding = 10) uniform sampler2D emissiveTexture;
-layout(set = 0, binding = 9) uniform sampler2D normalTexture;
+layout(set = 0, binding = 6) uniform sampler2D ambientTexture;
+layout(set = 0, binding = 5) uniform sampler2D diffuseTexture;
```

Comment:
Can't we have all these bindings #define-ed somewhere in one place?

Reply:
yeah lets add all bindings into separate file with #define Vulkan tag.